src/views/VenueDetails/VenueDescription/LocationMap/styles.module.css

```diff
@@ -0,0 +1,5 @@
+.mapContainer {
+    height: 500px;
+    width: 500px;
+    margin: 10px 0 10px 0;
+}
```

Comment:
new Line. Please set webstorm to add new line at the end of the file on save. Also you should set linter to point this out. I'm tired of pointing this out 😜